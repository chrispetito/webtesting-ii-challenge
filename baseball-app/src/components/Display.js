import React from "react";

export default function Display(props) {
  return (
    <div className="count-box">
      <div>
          {/* <h2 className='ball-header'>Balls</h2> */}
        <h1 className='count-number'>{props.count.balls}</h1>
      </div>

      <div>
          {/* <h2 className='strike-header'>Strikes</h2> */}
        <h1 className='count-number'>{props.count.strikes}</h1>
      </div>
    </div>
  );
}
