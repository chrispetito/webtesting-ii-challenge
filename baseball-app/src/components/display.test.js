import React from "react";
import { render } from "@testing-library/react";
import "@testing-library/react/cleanup-after-each";
import Display from "./Display";

describe('<Display />', () => {
    it('Display renders without crashing', () => {
        render(<Display />)
    })
    it('display matches snapshot', () => {
        const { container } = render(<Display />)
        expect(container).toMatchSnapshot();
    })
})