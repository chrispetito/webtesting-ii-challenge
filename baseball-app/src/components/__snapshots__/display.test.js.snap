// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Display /> display matches snapshot 1`] = `
<div>
  <div
    class="count-box"
  >
    <div>
      <h1
        class="count-number"
      >
        Balls: 
      </h1>
    </div>
    <div>
      <h1
        class="count-number"
      >
        Strikes: 
      </h1>
    </div>
  </div>
</div>
`;
